vars/applyTemplate.txt

```diff
@@ -1,30 +1,51 @@
 # applyTemplate
 
-This function processes template file with parameters file and then runs "oc apply" for the processing results.
-Also it has a workaround for the bug that if the DeploymentConfig already exists, it will break the Image Trigger.
+## Summary
```

Comment:
Shouldn't this fille be `applyTemplate.md` instead of `.txt`?

Reply:
you'd of thought so, but no :)
- https://jenkins.io/doc/book/pipeline/shared-libraries/#directory-structure

Reply:
Weird!